packages/fetchai/protocols/contract_api/custom_types.py

```diff
@@ -31,64 +28,4 @@
 RawMessage = BaseRawMessage
 RawTransaction = BaseRawTransaction
 State = BaseState
-
```

Comment:
Moved `Kwargs` to `aea.helpers.transaction.base` because it is now imported by multiple packages.

Reply:
In general this would be the right approach. But for this particular class, I prefer it if we have replication. The reason: "kwargs" class should really not be a class and just a dict with arbitrary primitive types inside. The problem: protocol generator does not yet support it. If we include it in the protocols, it's easier to upgrade in the future as we just change the protocol rather than an AEA class. So please just replicate across custom_types of the different protocols.